### PR TITLE
RFP: Wallet Aggregator Library

### PR DESCRIPTION
**Pull request by 0xCaso**
Maybe it could have a more decent name

### CODE REVIEW

File: docs/RFPs/Open/wallet-aggregator-library.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Project Description :page_facing_up: 
+
+Users of Polkadot and Substrate-based projects need to connect to the frontends with their wallets. At the moment, there are [several wallets and browser extensions](https://wiki.polkadot.network/docs/build-wallets) that can be used (Polkadot-JS, Talisman, Fearless, just to name a few). However, it's common that the frontends don't support all of them, and the users need to install a new wallet or browser extension to connect to the frontend.
+
+This project aims to create a **React library** that allows users to connect with any wallet or browser extension to the frontends that adopts it. This way, the users can use the wallet they prefer, and the frontends can support all of them without the need to implement the connection logic for each wallet, just integrating one library (making life easier for developers).
+
+## Deliverables :nut_and_bolt:
+
+The followings could be the initial deliverables of the project. Of course, improvements and additions are more than welcome.
```
Comment (keeganquigley) [inline docs/RFPs/Open/wallet-aggregator-library.md:14]:
```suggestion
The following items could be the initial deliverables of the project. Of course, improvements and additions are more than welcome.
```

File: docs/RFPs/Open/wallet-aggregator-library.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Wallet Aggregator Library
+
+* **Status:** Open 
+* **Proposer:** [Matteo Casonato](https://github.com/0xCaso)
+
+## Project Description :page_facing_up: 
+
+Users of Polkadot and Substrate-based projects need to connect to the frontends with their wallets. At the moment, there are [several wallets and browser extensions](https://wiki.polkadot.network/docs/build-wallets) that can be used (Polkadot-JS, Talisman, Fearless, just to name a few). However, it's common that the frontends don't support all of them, and the users need to install a new wallet or browser extension to connect to the frontend.
```
Comment (keeganquigley) [inline docs/RFPs/Open/wallet-aggregator-library.md:8]:
```suggestion
Users of Polkadot and Substrate-based projects need to connect their wallet to a front-end when using a dApp. At the moment, there are [several wallets and browser extensions](https://wiki.polkadot.network/docs/build-wallets) that can be used (Polkadot-JS, Talisman, Fearless, just to name a few). However, it's common that the frontends don't support all of them, and the users need to install a new wallet or browser extension to connect to the frontend.
```

File: docs/RFPs/Open/wallet-aggregator-library.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Wallet Aggregator Library
+
+* **Status:** Open 
+* **Proposer:** [Matteo Casonato](https://github.com/0xCaso)
+
+## Project Description :page_facing_up: 
+
+Users of Polkadot and Substrate-based projects need to connect their wallet to a front-end when using a dApp. At the moment, there are [several wallets and browser extensions](https://wiki.polkadot.network/docs/build-wallets) that can be used (Polkadot-JS, Talisman, Fearless, just to name a few). However, it's common that the frontends don't support all of them, and the users need to install a new wallet or browser extension to connect to the frontend.
+
+This project aims to create a **React library** that allows users to connect with any wallet or browser extension to the frontends that adopts it. This way, the users can use the wallet they prefer, and the frontends can support all of them without the need to implement the connection logic for each wallet, just integrating one library (making life easier for developers).
```
Comment (keeganquigley) [inline docs/RFPs/Open/wallet-aggregator-library.md:10]:
```suggestion
This project aims to create a **React library** that allows users to connect with any wallet or browser extension to the frontends that adopts it. This way, the users can use the wallet they prefer, and the frontends can support all of them without the need to implement the connection logic for each wallet, just integrating one library (making life easier for developers). Though we would prefer a React library, we would also consider implementations for other libraries as well.
```

**Comment by 0xCaso** [comment]
thanks a lot @keeganquigley!

**Review by keeganquigley — APPROVED** [review]
NP! LGTM

**Review by Noc2 — DISMISSED** [review]

**Review by dsm-w3f — DISMISSED** [review]

**Review by keeganquigley — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Review by dsm-w3f — APPROVED** [review]

**Comment by aka-toxa** [comment]
Hey team! 
It looks like there is a React solution for that called SubConnect: 

here is the video demo: https://bit.ly/38QhmfI
demo app: https://connect.subwallet.app
Github repo: https://github.com/Koniverse/SubConnect

I'm not related to the SubConnect team, but looking forward to sending a proposal for this RFP and want to understand the following: 

If something out there looks to me as something that covers your RFP, then what am I missing here? What needs to be covered so I can prepare a proposal to address the gap?

**Comment by semuelle** [comment]
Thanks for the info. I personally wasn't aware of SubConnect. Given SubConnect and [talisman-connect](https://github.com/TalismanSociety/talisman-connect), we might as well close the RFP or at least rewrite it to evaluate the "market" situation and shortcomings of existing solutions first. Would you be interested in such a project, @aka-toxa?

**Comment by aka-toxa** [comment]
hey, @semuelle I'm super interested in such projects. I have 10+ years of experience in software development and I feel that dev tools are my passion. I'm a part of big community of developers ([Magic-Powered](https://magicpowered.io)) who looking forward for new projects and partnerships

**Comment by Noc2** [comment]
Fyi: I just closed the RFP https://github.com/w3f/Grants-Program/commit/d43ef9bb31c5f39392b5461065b58311de6d6b28

**Comment by 0xslipk** [comment]
This is awesome. Thanks for the info @aka-toxa. We just open an [issue](https://github.com/Koniverse/SubConnect/issues/18) on SubConnect to add [Kuma](https://github.com/blockcoders/kuma-wallet)

**Pull request merged**